src/scripts/main.js

```diff
@@ -21,7 +21,23 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  if (!data || typeof data !== 'object' || Object.keys(data).length === 0) {
+    return null;
+  }
+
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (typeof data[key] === 'object' && data[key] !== null) {
+      createTree(li, data[key]);
```

Comment:
The recursive call to `createTree` should append the returned `ul` element to the `li` element. Currently, the function does not return anything, which means the `ul` is not being appended correctly. Modify the `createTree` function to return the `ul` element and append it to `li` here.